src/client/Microsoft.Identity.Client/MsalErrorMessage.cs

```diff
@@ -1,398 +1,398 @@
+
+        public const string MissingEntitlements =
+            "The application does not have keychain access groups enabled in the Entitlements.plist. " +
+            "As a result, there was a failure to save to the iOS keychain. " +
+            "The keychain access group '{0}' is not enabled in the Entitlements.plist. " +
+            "Also, use the WithIosKeychainSecurityGroup api to set the keychain access group. " +
+            "See https://aka.ms/msal-net-enable-keychain-groups for more details on enabling keychain access groups and entitlements.";
+
+        public const string AndroidActivityNotFound = "The Activity cannot be found to launch the given Intent. To ensure authentication, a browser with custom tab support " +
+            "is recommended. See https://aka.ms/msal-net-system-browsers for more details on using system browser on Android.";
+
+        public const string DefaultRedirectUriIsInvalid = "The OAuth2 redirect URI {0} should not be used with the system browser, because the operating system cannot go back to the app. Consider using the default redirect URI for this platform. See https://aka.ms/msal-client-apps for more details. ";
+
+        public const string RedirectUriContainsFragment = "'redirectUri' must NOT include a fragment component. ";
+
+        public const string NoRedirectUri = "No redirectUri was configured. MSAL does not provide any defaults. ";
+
+        public const string ClientApplicationBaseExecutorNotImplemented =
+            "ClientApplicationBase implementation does not implement IClientApplicationBaseExecutor. ";
+
+        public const string ActivityRequiredForParentObjectAndroid = "On Xamarin.Android, you have to specify the current Activity from which the browser pop-up will be displayed using the WithParentActivityOrWindow method. ";
+
+        public const string LoggingCallbackAlreadySet = "LoggingCallback has already been set. ";
+        public const string TelemetryCallbackAlreadySet = "TelemetryCallback has already been set. ";
+        public const string NoClientIdWasSpecified = "No ClientId was specified. ";
+        public const string AdfsNotCurrentlySupportedAuthorityType = "ADFS is not currently a supported authority type. ";
+        public const string TenantIdAndAadAuthorityInstanceAreMutuallyExclusive = "TenantId and AadAuthorityAudience are both set, but they're mutually exclusive. ";
+        public const string InstanceAndAzureCloudInstanceAreMutuallyExclusive = "Instance and AzureCloudInstance are both set but they're mutually exclusive. ";
+        public const string NoRefreshTokenProvided = "A refresh token must be provided. ";
+        public const string AadThrottledError = "Your app has been throttled by AAD due to too many requests. To avoid this, cache your tokens see https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/wiki/Client-Throttling.";

```

Comment:
Link should be aka.ms one - https://aka.ms/msal-net-throttling - (points to the same page).

Reply:
Good catch. Updated